skatepark/sass/blocks/_post-comments.scss

```diff
@@ -0,0 +1,169 @@
+@import "../../../blockbase/sass/blocks/button-mixins";
+
+.wp-block-post-comments {
+
+	line-height: var(--wp--custom--body--typography--line-height);
```

Comment:
if this is not the case by default, this is a good thing to do from Blockbase too

Reply:
Looks like this is the case by default actually, I've just tried removing it and it made no difference. It looks like it's applied to the body in the inline styles at the top of the page (the 'global-styles-inline-css' block).